### DISCUSSION TITLE
Problems in the training of mocov3

### DISCUSSION THREAD

**Discussion by majintao1994** [open-mmlab/mmselfsup#640] [How-to]
How to train mocoV3 with coco dataset？

**Comment by fangyixiao18** [comment]
Hi, here is an example to train MAE with COCO, you could follow this [page](https://github.com/open-mmlab/mmselfsup/discussions/511) to write your config

**Reply by majintao1994** [reply]
Some errors occurred，'mmdet.CocoDataset' is not  in the dataset  registry

**Reply by majintao1994** [reply]
run customdataset is ok

**Reply by fangyixiao18** [reply]
did you install mmdet?

**Reply by majintao1994** [reply]
yes,I did.I install  mmdet3.0.0rc4

**Reply by fangyixiao18** [reply]
what is the version of your other package? like mmengine, mmselfsup, mmcls

**Reply by majintao1994** [reply]
mmengine0.1.0, mmselfsup1.0.0rc2、mmcls1.0.0rc0

**Reply by majintao1994** [reply]
mmengine 0.1.0 、mmselfsup1.0.0rc2、mmcls1.0.0rc0

**Reply by fangyixiao18** [reply]
For mmselfsup1.0.0rc2, to use mmdet.CocoDataset, you need to add `custom_imports = dict(imports='mmdet.datasets', allow_failed_imports=False)`

In mmselfsup1.0.0rc3, we refactor this feature, the `custom_imports` is not needed, but it requires higher version mmcls and mmengine

**Reply by majintao1994** [reply]
ok,thinks for your reply
